packages/config/src/handlers/application/applicationConfigHandler.ts

```diff
@@ -63,9 +63,8 @@ async function getCertificates(request: Hapi.Request, h: Hapi.ResponseToolkit) {
 
   if (
     scope &&
-    (scope.includes(RouteScope.CERTIFY) ||
-      scope.includes(RouteScope.VALIDATE) ||
-      scope.includes(RouteScope.NATLSYSADMIN))
+    (scope.includes(SCOPES.RECORD_SUBMIT_FOR_APPROVAL) ||
+      scope.includes(SCOPES.CONFIG_UPDATE_ALL))
```

Comment:
Certificates are required by whomever can print? So `record.declaration-print` or `record.registration-print&issue-certified-copies`

Reply:
Makes sense. `record.declaration-print` is probably just for printing the record. So the latter one should be the one to go.